modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -51,11 +51,11 @@ let createQueryURL = (query, type) => {
 }
 
 // Recursive function to get a flat list of parents
-export function getHierarchy (node, RenderComponent, propsCreator) {
+export function getHierarchy (node, RenderComponent, propsCreator, includeSubjectLabel) {
```

Comment:
I think the code would be easier to understand if we rename this:

```suggestion
export function getHierarchy (node, RenderComponent, propsCreator, includeSubjectType) {
```